### PR TITLE
Readjust styles after milligram removal

### LINKED ISSUE #128: Remove Milligram package
In the early days of the app I added a package called Milligram, it's a lightweight UI framework. I was just using it for its super simple columns/row code while working on sidebar component. 

It's definitely not needed anymore and it's just adding extra weight to the app so it should be removed.  After it's removed, some of the styling will be affected and will need to be fixed.

### PR DESCRIPTION
**Pull request by digilou**
##Issue Number:
Closes #128 

## Description

##Do any new issues need to be created as a result of this PR"?
No.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

🚨Please review the [contribution guidelines](https://github.com/gwg-women/gwg-women-techmakers/wiki/Contribution-Guidelines) to this repository.
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] Created a new branch from 'dev' and PR is targeting the project's 'dev' branch.
- [X] The issue can be closed.

 Provide a descriptive branch name and once the pull request is approved and merged to dev, the branch can be deleted.

💔Thank you!

### CODE REVIEW

**Comment by sgluzins** [comment]
hi @digilou, just checked out your branch to test out the CSS and it looks awesome! There was one thing I just wanted to ask about but not sure if it was expected -

screen shot 2018-03-30 at 10 51 04 pm

When the screen is really small, the "SUBMIT" button is a bit cut off. Is this something that might relate to the PR/needs to be fixed?

**Comment by PepperAddict** [comment]
That may be because of my edit a while back in #105  @sgluzins :(  
When I gave @media screen I added a min-width: 
```
    .fullContainer {
        display: grid;
        min-width: 400px;
        grid-template-columns: 1fr;
    }
```
Knowing the problem it caused, it can be removed or adjusted.

File: src/css/index.scss [hunk 1]
```diff
@@ -62,103 +72,73 @@ h3 {
...
 
 /* This is the Search area */
 .searchContainer {
-    display: grid;
-    grid-column: span 2;
-    grid-row: span 1;
-    grid-column-start: 1;
+    grid-area: search;
+    padding: .8em;
+    align-items: flex-end;
```
Comment (PepperAddict) [inline src/css/index.scss]: My only critique is that I didn't like the stretched search bar post Milligram's removal. Maybe add a max-width: to a digit you'd prefer?

File: src/css/index.scss [hunk 2]
```diff
@@ -232,38 +208,19 @@ h3 {
         font-size: 1.1rem;
     }
     .fullContainer {
-        position:relative;
         display: grid;
         min-width: 400px;
```
Comment (PepperAddict) [inline src/css/index.scss]: the min-width was my doing and caused problems. Could you remove it for me or adjust it? I'd appreciate it.

**Review by PepperAddict — CHANGES_REQUESTED** [review]
I love the changes you've made and I've actually learned from it. 
My only critique is that the searchbar is stretched now after milligram's removal. 
The other suggestion is just a favor to fix my error.

**Comment by digilou** [comment]
@PepperAddict Take a look now. I'm going to be doing family stuff all day, so you if you see any other tweaks that need to be made, please feel free to jump into my branch and update it!

**Review by PepperAddict — APPROVED** [review]

**Review by sgluzins — APPROVED** [review]
wooo! looks good :)

**Pull request merged**